makefile.conf.in

```diff
@@ -331,6 +332,10 @@ endif
 LDFLAGS = $(ARCH_RTLDFLAGS) $(RTLDFLAGS)
 SYSLIBS = $(ARCH_RTLDFLAGS) -L$(LIBDIR) -lgen $(SYS_LDFLAGS)
 
+#Michel here. This might be the wrong way to do it. Please let
```

Comment:
Will OSC build and run for all platforms supported by RTcmix?
If so, this is fine.  Otherwise, it should be added to each supported platform's section, above, and removed from here.